src/js/utils.coffee

```diff
@@ -0,0 +1,22 @@
+_ = require 'lodash'
+
+exports.urlParams = ->
+  query = window.location.search.substring(1)
```

Comment:
We once had a minor problem using query parameters for static pages with the building models tool. It was either related to hosting on github pages or on cloudfront.  Sometimes the (?) used in the query param caused a 404 on the server end.  We ended up only using hash (#) based params. I wish I could remember more details.  This is fine for now. Just good to remember that in some edge case it may be an issue.